pylint/utils/utils.py

```diff
@@ -91,18 +91,18 @@ def tokenize_module(module):
         return list(tokenize.tokenize(readline))
 
 
-def _basename_in_blacklist_re(base_name, black_list_re):
+def _is_in_blacklist_re(path, black_list_re):
```

Comment:
Since you're renaming the function anyway, I suggest to also replace `blacklist` with `blocklist`.  See eg. section 1.2 in https://tools.ietf.org/html/draft-knodel-terminology-01.

Reply:
I think it better to stay consistent with the terminology used around and keep that work for a dedicated PR...